### DISCUSSION TITLE
“Bottom Center” manual level

### DISCUSSION THREAD

**Discussion by swbog** [Jyers/Marlin#141] [Q&A] [answered]
“Bottom center” to manual level menu. 

I’ve finally figured out how to compile and I’m geeked to name my printer daddy. 

Anyhow can I add another option to that menu and set coordinates? 

Reason being, I bought a three point leveler if I can do it I’ll install it. If not it goes into the f*** it bucket 

I see there’s three point bed leveling but will I miss out on grid mesh?

**Comment by Schnabulation** [comment, marked as answer]
I believe this is pretty simple: you would have to edit some code in this file: `Marlin/src/lcd/extui/lib/creality_dwin/creality_dwin.cpp`

Check out the lines 814 and following - there are the points defined for manual leveling.

**Reply by swbog** [reply]
Thank you for the direction! I was hopeful I’d be able to use coordinates to set my location but I’m still naive with all this. I’ll try and make sense of the commands and see if I can make it happen

oh wait I see x y spots maybe won’t be too bad. Maybe I should be able to copy paste lines and change values 

you think I’ll have to add an item to the list on lines 806-812

**Reply by Schnabulation** [reply]
...that, or just rewrite an existing one for now.

**Reply by swbog** [reply]
Good call! I might just take “center” for that. No adjustment to be had there anyway. Hey man thanks for all your help this is huge.

**Reply by Schnabulation** [reply]
No worries man, I hope it works for you!

**Reply by Schnabulation** [reply]
...and tell me how it went, ok?

**Reply by swbog** [reply]
Perfect! Both new button and replacing a button I wanted to try and both easily done and this is my first time tampering with anything like this. Will install the kit now!

**Reply by Schnabulation** [reply]
Good to hear, well done my man!
